pkg/model/deployment_source.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package model;
+
+option go_package = "github.com/pipe-cd/pipecd/pkg/model";
+
+message DeploymentSource {
```

Comment:
Note: repoDir is removed because it is only being used in the deploysource package. We will change our way to clone/prepare source later.